app/policies/application_policy.rb

```diff
@@ -13,23 +13,23 @@ def index?
   end
 
   def show?
-    scope.where(id: record.id).exists?
+    false
   end
```

Comment:
should at least still show it for admins

Reply:
I believe we should keep it false by default. We should ideally require that all policies are explicitly defined and not choose to fallback to ApplicationPolicy.

There are some cases where we don't want to show records to all admins, so false is the safest default.

Reply:
should we throw an error then? my original comment was because I was worried that we may have some of these left in production but I guess by returning false, we'll catch those pretty quickly.